### ISSUE TITLE
Synchronizing debtap-p64 database failed : `$_arch` is not set

### ISSUE THREAD

**Issue by tokoro10g** [helixarch/debtap-p64#2]
Hi, I got the following error messages when I executed `debtap-p64 -u` for the first time.

```
# debtap-p64 -u
==> Synchronizing pkgfile database...
:: Updating 6 repos...
  core is up to date
  extra is up to date
  multilib is up to date
  community is up to date
  archlinuxfr is up to date
  pnsft-pur is up to date
==> Synchronizing debtap-p64 database...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 Not Found

gzip: /var/cache/debtap/debian-packages-files.gz: unexpected end of file
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 Not Found

gzip: /var/cache/debtap/ubuntu-packages-files.gz: unexpected end of file
Synchronization failed. Exiting...
```

I think it's because `$_arch` is not set.
I don't have an idea which one is appropriate; `i386` or `amd64`, so can you fix this issue?

Thanks

**Comment by helixarch** [comment]
Fix in 2.8.2

**Comment by helixarch** [comment]
Fixed in 2.8.2
